templates/layout.html

```diff
@@ -1,19 +1,14 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Stripe</title>
-  <style type="text/css" media="screen">
-    form article label {
-      display: block;
-      margin: 5px;
-    }
-
-    form .submit {
-      margin: 15px 0;
-    }
-  </style>
+  <title>Mission Bit</title>
+  <link rel="shorcut icon" href="/static/favicon.ico" />
+  <link rel="stylesheet" type="text/css" href="/static/css/style.css" />
```

Comment:
the static folder is also missing from this branch, would explain why the files aren't getting served!